lib/src/chart/line_chart/line_chart.dart

```diff
@@ -9,10 +9,12 @@ import 'line_chart_painter.dart';
 
 class LineChart extends ImplicitlyAnimatedWidget {
   final LineChartData data;
+  final bool stickyTouchedIndicators;
```

Comment:
Removed in 228555066be793e81139a9ad6c52ba7f121e7aab
I will keep this in my local branch because I really need this. I think it would be really nice to have this since keeping the finger on a phone screen removes a lot of visibility. I think it's easier for the user if last touched item remains active without keeping the finger on the screen.

Reply:
Okay, then there should be a way to dismiss it, like touching outside the bars.

Reply:
Yes, it already works like this, if you touch outside the bars the touch is cleared.

Reply:
You were supposed to revert all changes about the sticky headers.

Reply:
Sorry for burdening an already complicated process but are we not doing the sticky touch indicators? Myself and multiple users all need this by default. I'd be happy to do a separate PR if it's necessary.

Reply:
No worries,
No, we are not going to have it, it should be handled out of the library.

Reply:
This is using internal methods. How can it be handled outside of the library?

Reply:
I have a much cleaner solution to this. Give mine a shot